bicep/main.bicep

```diff
@@ -1073,6 +1073,18 @@ param enableNodePublicIP bool = false
 
 param warIngressNginx bool = false
 
+@description('The name of the resource group to create the AKS cluster managed resources in')
```

Comment:
Since the resource group can't already exist, can you add the word **NEW** to the description to more strongly describe that this is the case.

Otherwise, looks good to me.

```output
rovisioning of resource(s) for container service aks-exisgtest in resource group gordons failed. Message: Node resource group 'gordonsmanaged' under subscription 'REDACTED' already exists, please provide a new one
```

Reply:
yep indeed RG cant be  pre-created.